### PR TITLE
Fix linter issues

### PR DESCRIPTION
**Pull request by lego963**

### CODE REVIEW

File: playbooks/lb_monitoring_setup.yml [hunk 1]
```diff
@@ -32,15 +32,30 @@
...
         ansible_ssh_private_key_file: "{{ local_private_key }}"
         ansible_ssh_common_args: >
+          -o UserKnownHostsFile=/dev/null
+          -o ProxyCommand='ssh -W %h:%p
+          -q linux@{{ controller_public_ip }}
+          -i {{ local_private_key }}
+          -o StrictHostKeyChecking=no
+          -o UserKnownHostsFile=/dev/null'
+          -o PasswordAuthentication=no
           -o ProxyCommand='ssh -W %h:%p -q linux@{{ controller_public_ip }}
```
Comment (anton-sidelnikov) [inline playbooks/lb_monitoring_setup.yml]: Why proxy command duplicated?

File: playbooks/lb_monitoring_setup.yml [hunk 2]
```diff
@@ -32,15 +32,30 @@
...
```
Comment (anton-sidelnikov) [inline playbooks/lb_monitoring_setup.yml]: Here

File: playbooks/dns_monitoring_setup.yml [hunk 3]
```diff
@@ -31,7 +31,16 @@
...
-        ansible_ssh_common_args: "-o ProxyCommand='ssh -W %h:%p -q linux@{{ controller_public_ip }} -i {{ local_private_key }}' -o PasswordAuthentication=no"
+        ansible_ssh_common_args: >
+          -o UserKnownHostsFile=/dev/null
+          -o ProxyCommand='ssh -W %h:%p
+          -q linux@{{ controller_public_ip }}
+          -i {{ local_private_key }}
+          -o StrictHostKeyChecking=no
+          -o UserKnownHostsFile=/dev/null'
+          -o PasswordAuthentication=no
+          -o ProxyCommand='ssh -W %h:%p -q linux@{{ controller_public_ip }}
```
Comment (anton-sidelnikov) [inline playbooks/dns_monitoring_setup.yml]: And here

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

**Comment by outcatcher** [comment]
Why this PR contains changes to `scripts/terraform-lint.sh`?

**Review by outcatcher — CHANGES_REQUESTED** [review]
Why this PR contains changes to scripts/terraform-lint.sh?

File: playbooks/lb_monitoring_setup.yml [hunk 4]
```diff
@@ -142,5 +157,5 @@
     elb_ip: "{{ hostvars.test_host.loadbalancer_public_ip }}"
     bastion_ip: "{{ hostvars.test_host.bastion_public_ip }}"
   tasks:
-  - name: Execute csm_test_utils  # noqa 301 305
-    shell: "nohup python3 -m csm_test_utils monitor --target {{ elb_ip }} --telegraf 'http://{{ bastion_ip }}' &"
+    - name: Execute csm_test_utils  # noqa 301 305
+      shell: "nohup python3 -m csm_test_utils monitor --target {{ elb_ip }} --telegraf 'http://{{ bastion_ip }}' &"
```
Comment (outcatcher) [inline playbooks/lb_monitoring_setup.yml]: Lint error, please use multi-line (`>`) string here

**Review by Polina-Gubina — DISMISSED** [review]

**Comment by lego963** [comment]
> Why this PR contains changes to scripts/terraform-lint.sh?

because script `terraform.sh` didn't work at all
`ubuntu-bionic | ./scripts/terraform-lint.sh: 6: Syntax error: "(" unexpected`

File: playbooks/lb_monitoring_setup.yml (hunk 1 above)
Comment (lego963) [reply playbooks/lb_monitoring_setup.yml]: I copied this from `devel` branch

**Comment by outcatcher** [comment]
> > Why this PR contains changes to scripts/terraform-lint.sh?
> 
> because script `terraform.sh` didn't work at all
> `ubuntu-bionic | ./scripts/terraform-lint.sh: 6: Syntax error: "(" unexpected`

Wow, that's strange. Seems the bash version differs from the one I've tested with. I will make a fix for that in #303

**Comment by lego963** [comment]
recheck

**Review by anton-sidelnikov — APPROVED** [review]

File: .zuul.yaml [hunk 5]
```diff
@@ -2,14 +2,12 @@
 - job:
     name: ansible-lint
     parent: otc-tox
-    voting: false
```
Comment (outcatcher) [inline .zuul.yaml:5]: Kets nake them voting in another PR

**Review by outcatcher — CHANGES_REQUESTED** [review]

File: .zuul.yaml (hunk 5 above)
Comment (lego963) [reply .zuul.yaml:5]: I have lint fixes, maybe here?

**Review by outcatcher — APPROVED** [review]

**Pull request merged**